node/package.json

```diff
@@ -24,6 +27,7 @@
   "author": "Lance Devs",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@neon-rs/cli": "^0.0.74",
```

Comment:
I agree. I hope neon-bindings gets an official tool for this soon. neon-rs is written by one of the core maintainers of neon-bindings, so I'm hopeful it will become official eventually. If that doesn't happen, we will have to decide whether we want to fork neon-rs and maintain it ourselves, or switch to napi-rs which does provide these utilities.